docs/pages/versions/unversioned/sdk/facebook.md

```diff
@@ -56,12 +56,12 @@ Your Facebook application ID. [Facebook's developer documentation](https://devel
 A map of options:
 
 - **permissions (_array_)** -- An array specifying the permissions to ask for from Facebook for this login. The permissions are strings as specified in the [Facebook API documentation](https://developers.facebook.com/docs/facebook-login/permissions). The default permissions are `['public_profile', 'email']`.
-- **behavior (_string_)** -- The type of login prompt to show. Currently this is only supported on iOS, and must be one of the following values:
+- **behavior (_string_)** -- The type of login prompt to show. Must be one of the following values:
 
-  - `'system'` (default) -- Attempts to log in through the Facebook account currently signed in through the device Settings. This will fallback to `native` behavior on iOS 11+ as Facebook has been removed from iOS's Settings.
+  - `'system'` (iOS default) -- Attempts to log in through the Facebook account currently signed in through the device Settings. This will fallback to `native` behavior on iOS 11+ as Facebook has been removed from iOS's Settings. iOS only.
   - `'web'` -- Attempts to log in through a modal `UIWebView` pop up.
   - `'browser'` -- Attempts to log in through Safari or `SFSafariViewController`.
-  - `'native'` -- Attempts to log in through the native Facebook app, but the Facebook SDK may fallback to `browser` instead.
+  - `'native'` (Android default) -- Attempts to log in through the native Facebook app, but the Facebook SDK may fallback to `browser` instead.
```

Comment:
How about moving `native` to under `system`, so the defaults are all at the top?

Reply:
Good call 👍